### ISSUE TITLE
Low validation accuracy on Something-Something V1 and Something-Something V2

### ISSUE THREAD

**Issue by niam06** [IBM/bLVNet-TAM#3]
Thanks for your excellent work!

I tried to train a bLVNet-101-TAM-a2-b4-f8x2 model on Something-Something V1 using the hyperparameters setting reflected in the following bash command:

CUDA_VISIBLE_DEVICES=0,1 python train.py --datadir /media/data/something_v1 --dataset st2stv1 -d 101 --groups 16 --logdir logs --lr 0.01 -b 64 --dropout 0.5 -j 36 --blending_frames 3 --epochs 50 --disable_scaleup --imagenet_blnet_pretrained

At the end of the training, I get only around 31% validation accuracy, whereas the training accuracy is around 79%. Comparing with TSM-ResNet50, the training accuracy is actually higher (79% vs 78%), so it is even more confusing on why the validation accuracy is that low. Is it overfitting by any chance? Or is there something wrong with preprocessing the validation data (note that I used the scripts you provided to preprocess data)? Any comments on what I may be doing wrong here? I am sharing the corresponding training/validation log.
[log.log](https://github.com/IBM/bLVNet-TAM/files/4186577/log.log)

Update: Trained bLVNet-101-TAM-a2-b4-f8x2 on Something-Something V2 as well. With the training script provided, validation accuracy reaches around 53% only (less than the ~60%  accuracy reported). Also, for Something-Something-V2, the training accuracy is lower compared to TSM-ResNet50 (88.2% with TSM vs. 78.5% with bLVNet). Please let me know your thoughts on what may be going wrong here. Log corresponding to SthV2 training - 
[log.log](https://github.com/IBM/bLVNet-TAM/files/4205020/log.log)

**Comment by chunfuchen** [comment]
sorry, I do not notice the issue, I will take a look.

**Comment by chunfuchen** [comment]
May I know that what is the accuracy on Something-Something V1 and V2 if you used the provided model?

**Comment by ilkarman** [comment]
@niam06 I was wondering if you managed to get higher val acc since you posted? I seem to have similar issues with Kinetics.
